x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/190_lookup_join.yml

```diff
@@ -37,8 +51,14 @@ setup:
         body:
           actions:
             - add:
-                index: test-lookup
+                index: test-lookup-1
                 alias: test-lookup-alias
+            - add:
+                index: test-lookup-*
+                alias: test-lookup-alias-pattern-multiple
+            - add:
+                index: test-lookup-1*
```

Comment:
Is `*` intentional here? I think it is better to avoid it to ensure only a single index could match the alias